### DISCUSSION TITLE
监听复制节点

### DISCUSSION THREAD

**Discussion by paiap** [didi/LogicFlow#1199] [Q&A] [answered]
我现在需要监听用户通过ctrl+v或者右键复制，复制出来的节点我还需要进一步处理，请问有没有内置函数可以监听到节点复制的函数，并可以获取到复制节点的nodes呀？

**Comment by towersxu** [comment]
你可以参考源码中的ctrl+v功能，自定义此功能。

**Reply by paiap** [reply]
能告知以下一下大概在哪吗，谢谢！！！

**Comment by towersxu** [comment, marked as answer]
首先看自定义快捷键文档：https://site.logic-flow.cn/docs/#/zh/guide/basic/keyboard
然后把参考内置的自定义功能进行修改。https://github.com/didi/LogicFlow/blob/389067969588fd2766408b533eb27b50946bab9d/packages/core/src/keyboard/shortcut.ts#L61-L74

**Reply by paiap** [reply]
好的，非常感谢！！！
